RespiratoryAssistDevices/R4/resources/Questionnaire-R4-RespiratoryAssistDevicesLab.json

```diff
@@ -0,0 +1,789 @@
+    ]
+  },
+  "title": "Respiratory Assist Device Laboratory Test Results",
+  "status": "draft",
+  "subjectType": [
+    "Patient"
+  ],
+  "date": "2020-07-08",
+  "publisher": "Da Vinci DTR",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/RespiratoryAssistDevices-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicClinicalInfo-prepopulation"
+    }
+  ],
+  "item": [
+    {
+      "linkId": "1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/patient-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "2",
+      "type": "group",
+      "text": "Provider (physician/NPP) who is performing the face-to-face evaluation:",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/questionnaire-item-control",
+                "code": "gtable"
+              }
+            ]
+          }
+        }
+      ],
+      "item": [
+        {
+          "linkId": "2.1",
+          "text": "Last Name",
+          "type": "text"
+        },
+        {
+          "linkId": "2.2",
+          "text": "First Name",
+          "type": "text"
+        },
+        {
+          "linkId": "2.3",
+          "text": "MI",
+          "type": "text"
+        },
+        {
+          "linkId": "2.4",
+          "text": "NPI",
+          "type": "text"
+        },
+        {
+          "linkId": "2.5",
+          "text": "Date of F2F encounter",
+          "type": "date",
+          "required": true,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"RespiratoryAssistDevicesPrepopulation\".EncounterDate"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "3",
+      "text": "Person or entity performing testing",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "3.1",
+          "type": "text",
+          "text": "Laboratory",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"RespiratoryAssistDevicesPrepopulation\".TestLaboratory"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "3.2",
```

Comment:
Lab's NPI can also be pre populated from Organization resource.

Reply:
The same as above as we can't retrieve the observation's organization as it is not related to the patient.